### PR TITLE
#2121 CsvAppFileIO changed to use file system and process services

### PR DESCRIPTION
**Pull request by karthiknadig**
CsvAppFileIO changed to use IApplicationShell, IFileSystem, and IProcessServices

### CODE REVIEW

**Comment by MikhailArkhipov** [comment]
Usually this is only needed if code becomes testable but since it is calling GetGlobalService and GetExportedValue, it is still not :-)

But OK.

**Comment by MikhailArkhipov** [comment]
👍

File: src/Package/Impl/DataInspect/Office/CsvAppFileIO.cs [hunk 1]
```diff
@@ -28,68 +30,67 @@ internal static class CsvAppFileIO {
...
             var variableName = result.Name ?? _variableNameReplacement;
-            var csvFileName = MakeCsvFileName(variableName);
+            var csvFileName = MakeCsvFileName(appShell, pss, variableName);
 
-            var pss = VsAppShell.Current.ExportProvider.GetExportedValue<IProjectSystemServices>();
             var file = pss.GetUniqueFileName(folder, csvFileName, "csv", appendUnderscore: true);
 
             string currentStatusText;
-            var statusBar = VsAppShell.Current.GetGlobalService<IVsStatusbar>(typeof(SVsStatusbar));
+            var statusBar = appShell.GetGlobalService<IVsStatusbar>(typeof(SVsStatusbar));
```
Comment (AlexanderSher) [inline src/Package/Impl/DataInspect/Office/CsvAppFileIO.cs]: Is it VS status bar, or it is a status bar of some dialog window?

Comment (MikhailArkhipov) [inline src/Package/Impl/DataInspect/Office/CsvAppFileIO.cs]: VS status bar

Comment (AlexanderSher) [inline src/Package/Impl/DataInspect/Office/CsvAppFileIO.cs]: Hm... probably it is not part of this change, but since we have `IStatusBar`, probably we should use it instead. That will allow to move this code to the `Components` assembly.

**Comment by karthiknadig** [comment]
Created a work item to make the changes suggested by @MikhailArkhipov and @AlexanderSher.
